pygmt/clib/session.py

```diff
@@ -493,15 +495,11 @@ def get_default(self, name):
         )
 
         # Make a string buffer to get a return value
-        value = ctp.create_string_buffer(10000)
-
+        value = ctp.create_string_buffer(4096)
```

Comment:
The `GMT_Get_Default` API function only says that `value` must have enough space to store the return value but doesn't say the exact length, but function [`gmtlib_getparameter`](https://github.com/GenericMappingTools/gmt/blob/9d48c37ef02882fd8f97673f4da35250f8c6c05f/src/gmt_init.c#L12631) (called by `GMT_Get_Default`) defines a `GMT_BUFSIZ`-long (4096) char. 

So, we can safely change 10000 to 4096 in this function.